app/components/PerusePages/Error/Error.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Error extends Component
+{
+    static propTypes =
+    {
+        error : PropTypes.shape( {
+            header    : PropTypes.string,
+            subHeader : PropTypes.string
+        } )
+    }
+
+    static defaultProps =
+    {
+        error : { header: '', subHeader: '' }
+    }
+
+    render( )
+    {
+        const { error } = this.props;
+        const pageStyle = {
```

Comment:
@hunterlester can we align more with the other 'peruse pages', to use `Page` `Header` components etc (as makes sense), and keep the CSS in a separate file.

Reply:
@joshuef Right, that's what I would prefer as well.
When I attempt to import nessie components into the `Error` component, I receive the following upon `yarn mock-dev`:
```
ReferenceError: navigator is not defined
    at C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\node_modules\codemirror\lib\codemirror.js:23:15
    at e (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\node_modules\codemirror\lib\codemirror.js:18:5)
    at Object.module.exports.Object.defineProperty.value (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\node_modules\codemirror\lib\codemirror.js:22:17)
    at n (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\webpack\bootstrap:37:1)
    at e.defineMode.e.getMode.name (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\node_modules\codemirror\mode\jsx\jsx.js:8:5)
    at Object.<anonymous> (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\node_modules\codemirror\mode\jsx\jsx.js:28:14)
    at Object.module.exports.e.exports.default (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\src\CodeEditor\codeEditor.css?9e69:2:19)
    at n (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\webpack\bootstrap:37:1)
    at Object.module.exports.Object.defineProperty.value (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\src\CodeEditor\index.jsx:133:18)
    at n (C:\Users\guilf\safe\dev\browser\node_modules\nessie-ui\dist\webpack:\webpack\bootstrap:37:1)
```
Must be something do to with importing resources in the Express server route, similar to how I wasn't able to utilize `ejs` as a view engine.